### PR TITLE
Third homework

### PR DESCRIPTION
**Pull request by ghost**
Could you, please, check my home task №3

### CODE REVIEW

File: yGavrilovets/01_HTML/HW1_Python1.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<HTML
```
Comment (pluhin) [inline yGavrilovets/01_HTML/HW1_Python1.html]: missing >

File: yGavrilovets/02_HTML/Python/HW1_Python1.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<HTML
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Python_1</TITLE>
+	</HEAD>
+	<BODY>
+		<TABLE width="100%">
```
Comment (pluhin) [inline yGavrilovets/02_HTML/Python/HW1_Python1.html]: could you please remove this table, if you want to align picture, then use align attribute here, make changes in all same files with this logo-picture

File: yGavrilovets/02_HTML/Python/HW1_Python1.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+greg@earth:~/python_examples$</code></pre>
+		<P>Вот и всё.</P>
+		<H2 id="cycle">Пример цикла For</H2>
+		<P><a href="#contents">К содержанию<a></P>
+		<P>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый редактор и наберите следующую программу:</P>
+		<pre><code>#!/usr/bin/env python
+for cntr in range(0,10):
+print cntr</code></pre>
+		<P><em>Не забудьте поставить символ табуляции перед строкой «print cntr».</em> Это важно. В Python для указания на блоки кода используются не круглые «(» и фигурные «{» скобки как в других языках программирования, а отступы.</P>
+		<TABLE width="100%">
```
Comment (pluhin) [inline yGavrilovets/02_HTML/Python/HW1_Python1.html]: Why two tables?

File: yGavrilovets/03_HTML/basicForm.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+					</FORM>
+				</TD>
+			</TR>
+			<TR>
+				<TD>
+					<H3>Простая форма авторизации:</H3>
+					<FORM id="3" name="authorization" method="post" action="http://sc.it-academy.by/fe/form.php">
+						<FIELDSET>
+							<LEGEND>Авторизируйтесь</LEGEND>
+							<INPUT type=""text" id="login" name="login" required><span>*</span>
```
Comment (pluhin) [inline yGavrilovets/03_HTML/basicForm.html]: duplicate "

File: yGavrilovets/03_HTML/basicForm.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+						</FIELDSET>
+					</FORM>
+				</TD>
+				<TD>
+					<H3>Валидация данных HTML5:</H3>
+					<FORM id="8" name="validation" method="get" action="http://sc.it-academy.by/fe/form.php">
+						<FIELDSET>
+							<LEGEND>Строка поиска</LEGEND>
+							<INPUT type="text" pattern="[А-Яа-яЁё]{0,100}" name="russianText" placeholder="Только русские буквы"><br>
+							<INPUT type="tel"  pattern="[+]375-29-[0-9]{7}" name="phoneNumber" placeholder="+375-29-1234567"><br>
```
Comment (pluhin) [inline yGavrilovets/03_HTML/basicForm.html]: Wrong pattern, could you please check task requirements

**Review by pluhin — CHANGES_REQUESTED** [review]

File: yGavrilovets/02_HTML/Python/HW1_Python1.html (hunk 3 above)
Comment (ghost) [reply yGavrilovets/02_HTML/Python/HW1_Python1.html]: one table is for buttons, done with .img files. second table is for generated .svg files from additional task. i decided not 
to insert code for svg images in main file and pasted it in single files. 

 or should i make one table with 2 rows?

File: yGavrilovets/05_HTML/list.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<HTML>
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Список</TITLE>
+		<STYLE>
+		.list{width:350px}
+		UL{list-style:upper-roman}
+		LI UL LI{list-style:lower-roman}
+		.code{width:400px; color:green; background-color:black; max-height: 4em; overflow: auto}
```
Comment (pluhin) [inline yGavrilovets/05_HTML/list.html:10]: here better to use rem

**Review by pluhin — APPROVED** [review]

**Pull request merged**